config.json

```diff
@@ -95,19 +52,18 @@
       ]
     },
     {
-      "slug": "simple-cipher",
-      "uuid": "62d60b42-93bc-4de9-90d1-1ca18a847812",
+      "slug": "bob",
+      "uuid": "a5bf36f0-5d3c-41d4-8d54-e37e484e59cd",
       "core": true,
       "unlocked_by": null,
-      "difficulty": 1,
+      "difficulty": 2,
       "topics": [
-        "algorithms",
         "control_flow_conditionals",
-        "control_flow_loops",
-        "randomness",
+        "pattern_recognition",
```

Comment:
We don't have a specific set of guidelines, but those are an awful lot of topics :) I would suggest perhaps only listing the "core" ones. For example, it now lists `"regular_expressions"` as a topic, but is that something a student necessarily has to use when doing the exercise? If not, it might be useful to drop it (the C# track is not a great example by the way, as the topics there should be cleaned-up for the exact same reason).

Reply:
Yeh I want to retag them anyway because they are also wrong sometimes!